plugins/qcheck-stm/test/dune

```diff
@@ -1,3 +1,6 @@
+(cram
+ (deps %{bin:ortac}))
```

Comment:
This should take the complete set of dependencies (`package ...`) as other tests and be attached to the qcheck-stm package.

Reply:
github CI is indeed complaining when trying to install the packages.